config/routes.rb

```diff
@@ -7,6 +7,7 @@
   resources :landing, only: [:index]
 
   get 'landing/hpi', to: 'landing#hpi'
+  get 'privacy-notice', to: 'privacy#index'
```

Comment:
I didn't know whether to add either a leading 'landing/' or 'doc/' for this route, it's not immediately clear to me what the logical criteria is/was for them both, and whether the privacy policy should be grouped with one, the other, or standalone like this.

Reply:
I suspect the `landing/hpi` route is never used any more. HPI was the fore-runner to UKHPI, which got a complete ground-up rewrite when we migrated to the country-wide index. I could, however, see an argument for putting `doc/privacy-notice` alongside `doc/accessibility`.

Reply:
I think that seems correct, for consistency: `doc/privacy-notice`.

Should the `landing/hpi` route be removed?

Reply:
If memory serves, the `landing/hpi` route does a redirect to UKHPI, so I think it should stay.